app/posts/manage-teacher-training-applications/2020-05-04-setting-up-permissions-iteration-3.md

```diff
@@ -0,0 +1,185 @@
+description: Let providers set up permissions between themselves and their partner organisations
+date: 2020-05-04
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Previous iterations:
+
+* [first iteration](/manage-teacher-training-applications/setting-up-permissions)
+* [second iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2)
+
+In this iteration we improved the design and tested it with a user who works at Somerset SCITT who:
+
+* run and ratify their own courses
+* ratify courses for Growing Expert Teachers, Taunton Teaching Alliance, The Beach Teaching School and Ventrus Teaching School Alliance.
+
+The prototype is based on the user's access on Publish which shows they belong to Ventrus Teaching School Alliance. During the session we learnt that the user is also belong of the other 3 organisations too.
+
+This means that while this person works for Somerset SCITT, these training providers delegate power and control to people outside of their organisation.
+
+This set up is quite common across other partnerships on Apply.
+
+The journeys include:
+
+- setting up permissions for partner organisations for the first time
+- inviting a new user to their organisation
+- managing organisational permissions and users
+
+## Hypotheses
+
+### Showing the user belongs to multiple organisations
+
+The [previous design](/manage-teacher-training-applications/setting-up-permissions-iteration-2) didn’t handle the situation where a user belongs to multiple organisations.
+
+If we show the user they're part of multiple organisations
+Then they'll understand what’s about to happen on the following screens
+
+{{ appFigure({
+  image: {
+    file: "before-you-start.png"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "data-sharing.png"
+  }
+}) }}
+
+### Making sure users understand permissions
+
+In the [previous iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2) we had information about permissions on each page that lets uses select permissions for each organisation.
+
+But there’s a lot of information to digest and we want to be sure users understand their meaning and implications.
+
+If we put this information on a dedicated page before they select permissions
+Then users will have the best chance of understanding how permissions work
+
+{{ appFigure({
+  image: {
+    file: "step2.png"
+  }
+}) }}
+
+### Making it easier to understand permissions when setting up permissions between organisations
+
+In the [previous iteration](/manage-teacher-training-applications/setting-up-permissions-iteration-2) we added information about permissions on each page that contains permission checkboxes.
+
+Each permission type is collapsed inside a Details panel. This is a little messy and will get messier when new permissions are added.
+
+If we put the information behind a single panel
+Then the user will have less to scan through
+
```

Comment:
Details in a box is a bit 🤨

Reply:
Wondering if not using indented text for the disclosed content would improve this?

Reply:
@fofr what's would you do instead of a box here. I tried first without a box and it's quite messy and lost amongst the h1 and form controls.

@paulrobertlloyd is this what you mean:

![image](https://user-images.githubusercontent.com/37163/80988023-d023d000-8e2a-11ea-9d6d-d3fa058ba23b.png)

I guess my (minor) concern is going against the pattern which in this case makes it less clear the thing is now open.